netci.groovy

```diff
@@ -1897,7 +1897,7 @@ def static calculateBuildCommands(def newJob, def scenario, def branch, def isPR
 
                             if (enableCorefxTesting) {
                                 def fxRoot = "%WORKSPACE%\\_\\fx"
```

Comment:
It's not that that path is horked, it's that when we validate the path, we're checking that the path already exists, which it of course doesn't yet. The only other option I can think of is to not check that that directory exists at all.